### ISSUE TITLE
Question: How does the scheduled check for updates work exactly?

### ISSUE THREAD

**Issue by RavenMacDaddy** [martinet101/WingetUI#222]
I've set the app to automatically check for updates every 4 hours (instead of 1 hour), basically to not get notifications too frequently for my tastes.

Though, how does the app count and execute that? For example, what if the computer goes to sleep after 30 mins of inactivity, and let's say for this example that I've been using the computer for 1 hour before that.

After the PC waking up, will the app continue counting from the 1.5 hours that have passed up to the set 4 – or would that mean that the app erroneously starts from 0 again? 

In other words, requiring the system to be on for 4 hours straight, in this case. 

The question got stuck in my mind as a result of issue #221.

**Comment by martinet101** [comment]
> After the PC waking up, will the app continue counting from the 1.5 hours that have passed up to the set 4 – or would that mean that the app erroneously starts from 0 again?

Honestly, I don't know, that depends on pythn's time.sleep() function.

What it does is, after checking for updates, start a Thread that will wait x seconds (depending on what is set on the settings window), and then update sources and run the update checker.

**Comment by RavenMacDaddy** [comment]
Fair enough – thanks for the input and clarification so far.
